lbrynet/core/Wallet.py

```diff
@@ -975,6 +976,9 @@ def _update_balance(self):
             lambda result: Decimal(result['confirmed']) + Decimal(result.get('unconfirmed', 0.0)))
         return d
 
+    def get_max_usable_balance_for_claim(self, claim_name):
```

Comment:
Please add a matching dummy method in `Wallet` too

Reply:
Done!